lib/generators/shopify_app/home_controller/home_controller_generator.rb

```diff
@@ -29,16 +27,26 @@ def add_home_index_route
 
       private
 
+      def embedded?
+        options['embedded'] == 'true'
+      end
+
       def embedded_app?
-        ShopifyApp.configuration.embedded_app? || options['embedded'] == 'true'
```

Comment:
Thinking about this from the CLI's perspective, we may want to still consider the option here so we can control it from the command line (even though I don't think the CLI currently supports it).

Reply:
@paulomarg so this is how you would pass in the options now:
**default embedded app**: `rails generate shopify_app`
**default non-embedded app (uses cookies)**: `rails generate shopify_app --embedded false`
**embedded app with cookies**: `rails generate shopify_app --with-cookie-authentication`

Is this what you meant?

Reply:
Yes, as long as we can control this from the command line I'm happy! I thought we were dropping the `embedded` option altogether, and once we've ironed out all of these choices we should change the CLI to ask whether the app is embedded or not.

Reply:
Not sure about removing the `embedded` option as I'm not sure how else we would specify that an app is non-embedded? If it is to be removed, I think that should be part of a separate PR.

Reply:
Sorry, I wasn't super clear there. I'd like for the flag to stay, I thought before that you were actually removing it, so my mistake!

I think the options you suggested above sound perfect - and non-embedded + cookie is definitely not recommended 👍